pkgs/desktops/gnome/core/libgnome-keyring/default.nix

```
@@ -13,9 +13,23 @@ stdenv.mkDerivation (finalAttrs: {
 
   outputs = [ "out" "dev" ];
 
+  strictDeps = true;
   propagatedBuildInputs = [ glib gobject-introspection dbus libgcrypt ];
   nativeBuildInputs = [ pkg-config intltool ];
 
+  configureFlags = [
+    # not ideal to use -config scripts but it's not possible switch it to pkg-config
+    # binaries in dev have a for build shebang
```

I don't understand the end of this sentence, but for sure this word is missing:

```suggestion
    # not ideal to use -config scripts but it's not possible to switch it to pkg-config
```

---

Same here, I just cherry-picked @Artturin’s commit and gave up on it.

I believe it is supposed to be parsed “binaries in dev have (a (for build) shebang)”

---

> Same here, I just cherry-picked @Artturin’s commit and gave up on it.
> 
> I believe it is supposed to be parsed “binaries in dev have (a (for build) shebang)”

@Artturin gave their :+1: for rephrasing that sentence as you said.